tools/devtool

```diff
@@ -616,6 +731,22 @@ cmd_help() {
     echo ""
 }
 
+help_release() {
```

Comment:
Could we also print an example of how this command is run here, like the one below (`Example: devtool prepare_release 0.42.0`)?

Reply:
This is not a command that can be called by the user it is just an auxiliary function that is used to print the help menu for the `release` command.